### ISSUE TITLE
ios build errors after updating to Mac Catalina and xcode 10.2

### ISSUE THREAD

**Issue by ajgomezrivera** [ionic-team/capacitor#2122]
I updated my mac to Catalina OS and xcode 10.2 and now when I try to build an ionic app the process fails. The errors are the ones shown in the image:
![Screen Shot 2019-10-31 at 9 57 32 AM](https://user-images.githubusercontent.com/6227710/67953348-508c1e80-fbc5-11e9-859b-86321f1272a2.png)

Can anyone please guide me to the necessary solution?

The package.json files has:
{
  "name": "myApp",
  "version": "0.0.1",
  "author": "Ionic Framework",
  "homepage": "https://ionicframework.com/",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e"
  },
  "private": true,
  "dependencies": {
    "@angular/common": "~8.1.2",
    "@angular/compiler": "~8.1.2",
    "@angular/core": "~8.1.2",
    "@angular/forms": "~8.1.2",
    "@angular/platform-browser": "~8.1.2",
    "@angular/platform-browser-dynamic": "~8.1.2",
    "@angular/router": "~8.1.2",
    "@capacitor/core": "1.2.1",
    "@capacitor/ios": "^1.2.1",
    "@ionic-native/core": "^5.0.0",
    "@ionic-native/splash-screen": "^5.0.0",
    "@ionic-native/status-bar": "^5.0.0",
    "@ionic/angular": "^4.7.1",
    "core-js": "^2.5.4",
    "rxjs": "~6.5.1",
    "tslib": "^1.9.0",
    "zone.js": "~0.9.1"
  },
  "devDependencies": {
    "@angular-devkit/architect": "~0.801.2",
    "@angular-devkit/build-angular": "~0.801.2",
    "@angular-devkit/core": "~8.1.2",
    "@angular-devkit/schematics": "~8.1.2",
    "@angular/cli": "~8.1.2",
    "@angular/compiler": "~8.1.2",
    "@angular/compiler-cli": "~8.1.2",
    "@angular/language-service": "~8.1.2",
    "@capacitor/cli": "1.2.1",
    "@ionic/angular-toolkit": "~2.0.0",
    "@types/jasmine": "~3.3.8",
    "@types/jasminewd2": "~2.0.3",
    "@types/node": "~8.9.4",
    "codelyzer": "^5.0.0",
    "jasmine-core": "~3.4.0",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "~4.1.0",
    "karma-chrome-launcher": "~2.2.0",
    "karma-coverage-istanbul-reporter": "~2.0.1",
    "karma-jasmine": "~2.0.1",
    "karma-jasmine-html-reporter": "^1.4.0",
    "protractor": "~5.4.0",
    "ts-node": "~7.0.0",
    "tslint": "~5.15.0",
    "typescript": "~3.4.3"
  },
  "description": "An Ionic project"
}

I have tried clean build, removing old files, removing ios and creating ios folder again with ionic capacitor.

The project I'm actually working on is not the one shown here, but I've been working fine in other projects until I did the upgrade to Catalina and xcode 10.2, from Sierra and xcode 8.3.3

**Comment by oliverandersencox** [comment]
same here

**Comment by oliverandersencox** [comment]
I can't even get to that point. Npx cap sync gives me:

>  Error running update: /bin/sh: /usr/local/bin/pod: /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/bin/ruby: bad interpreter: No such file or directory

**Comment by ajgomezrivera** [comment]
I was able to build my project after trying lots of things. What seems to have solved it was: 

Did a clean install of xcode (I now installed xcode 11.2) and cocoapods:
-Uninstalled cocoapods and clean all related files
https://superuser.com/questions/686317/how-to-fully-uninstall-the-cocoapods-from-the-mac-machine

- Uninstalled xcode 10.2 and cleaned all related files
https://macpaw.com/how-to/uninstall-xcode-on-macos

Then I reinstalled both. Now I'm able to build my ionic project for simulator!
